src/main/kotlin/baseball/Game.kt

```diff
@@ -0,0 +1,43 @@
+package baseball
+
+import baseball.Constant.CORRECT_THREE_NUMBERS_MESSAGE
+import baseball.Constant.FINISH_BASEBALL_GAME_MESSAGE
+import baseball.Constant.START_BASEBALL_GAME_MESSAGE
+import baseball.Constant.THREE_STRIKE
+import baseball.Constant.ZERO_BALL
+
+object Game {
+    fun start() {
+        println(START_BASEBALL_GAME_MESSAGE)
+        startBaseballGame()
+    }
+
+    private fun startBaseballGame() {
+        Computer.setNumber()
+        val computerNumber = Computer.getNumber()
+        do {
+            GameCount.resetCount()
+
+            Computer.askNumber()
+            Player.inputNumber()
+            val playerNumber = Player.getNumber()
+
+            Hint.compareNumber(playerNumber, computerNumber)
+            val count = GameCount.getCount()
+            Hint.printHint(count)
+        } while (!isAnswer())
+        // 정답을 맞춘 후 실행됨
+        println(CORRECT_THREE_NUMBERS_MESSAGE + FINISH_BASEBALL_GAME_MESSAGE)
+        Computer.askRestartOrFinish()
+        Player.inputRestartOrEnd()
+    }
+
+    fun reStart() {
+        startBaseballGame()
+    }
+
+    private fun isAnswer(): Boolean {
+        val (ball, strike) = GameCount.getCount()
+        return (ball == ZERO_BALL && strike == THREE_STRIKE)
```

Comment:
게임 종료 조건이 모두 스트라이크인 경우 게임이 종료되므로
THREE_STRIKE 라는 이름을 MAX_STRIKE라고 변경 후 아래와 같이 작성하시면 ball == ZERO_BALL를 연산할 필요가 없지 않나 싶습니다.
```suggestion
        return strike == MAX_STRIKE
```

Reply:
좋은 생각인 것 같습니다 ! 3스트라이크면 볼은 당연이 0이므로 연산할 필요가 없을 것 같네요 ! 의견 공유 감사합니다.